src/data/content/design-system/componenti/buttons.yaml

```diff
@@ -269,7 +277,13 @@ tabs:
             title: Comportamento
             headingLevel: 2
             text: |
-              *In stesura*
+              Elenco degli **stati interattivi** disponibili in tutte le varianti del componente:
+
+              - **default**, il pulsante si presenta nello stato normale quando è attivo ma l'utente non ci interagisce;
```

Comment:
@danielaiozzo anche qui preferisci usare i `:`al posto della `,`? Fai tu le suggestions in caso?

Reply:
sì, ma dovresti avere le suggestion dei commit precedenti...